python/ray/train/torch/train_loop_utils.py

```diff
@@ -63,11 +69,64 @@ def get_device() -> Union[torch.device, List[torch.device]]:
         >>> # ray.get_gpu_ids() == [4,5]
         >>> # torch.cuda.is_available() == True
         >>> # get_device() == torch.device("cuda:4")
+
+
+        >>> # You can move model to device by:
+        >>> # model.to(ray.train.torch.get_device())
+        >>> #
+        >>> # instead of manually checking the device type:
+        >>> # model.to("cuda" if torch.cuda.is_available() else "cpu")
     """
```

Comment:
Can we remove the "comments" for the text so that it renders as regular text? See [ray.data.read_csv](https://docs.ray.io/en/master/data/api/doc/ray.data.read_csv.html) as an example of what I mean.

Reply:
Ok. Updated!